tests/DefaultStandardTypeRegistryTest.php

```diff
@@ -0,0 +1,55 @@
+<?php declare(strict_types=1);
+
+namespace GraphQL\Tests;
+
+use GraphQL\Error\InvariantViolation;
+use GraphQL\Type\Definition\BooleanType;
+use GraphQL\Type\Definition\FloatType;
+use GraphQL\Type\Definition\IDType;
+use GraphQL\Type\Definition\IntType;
+use GraphQL\Type\Definition\StringType;
+use GraphQL\Type\Registry\DefaultStandardTypeRegistry;
+use PHPUnit\Framework\TestCase;
+
+final class DefaultStandardTypeRegistryTest extends TestCase
+{
+    private DefaultStandardTypeRegistry $registry;
+
+    protected function setUp(): void
+    {
+        parent::setUp();
+        $this->registry = new DefaultStandardTypeRegistry();
+    }
+
+    /** @throws InvariantViolation */
```

Comment:
Not necessary, see https://github.com/webonyx/graphql-php/blob/0a917058620a197530b357c46d8d5943a054a37e/phpstan.neon.dist#L50-L53